tests/common/devices.py

```diff
@@ -312,6 +312,28 @@ def get_networking_uptime(self):
             self.logger.error("Exception raised while getting networking restart time: %s" % repr(e))
             return None
 
+    def get_image_info(self):
+        """
+        @summary: get list of images installed on the dut.
+                  return a dictionary of "current, next, installed_list"
+        """
+        output = self.command("sonic_installer list")["stdout"]
+        lines  = output.split('\n')
+        ret    = {}
+        list   = []
```

Comment:
`list` is a keyword - maybe `image_list`?

Reply:
In fact, there is another field that has stdout as lines, it is:

lines = self.command("sonic_installer list")["stdout_lines"]

Reply:
the return value is documented in shell command:

https://docs.ansible.com/ansible/latest/modules/shell_module.html#return-values

Reply:
thanks. changed!

Reply:
Changed according to Tamer's suggestions.